controllers/CommentController.js

```diff
@@ -0,0 +1,17 @@
+const Comment = require('../models/Comment');
+
+
+const createComments = async (req , res) =>{
+    const { comment } = req.body
+    if(!comment){
+        res.status(422).json({error:"Please write a comment"})
+    } 
+
+  const newcomment =  new Comment({comment , postBy: req.user})
```

Comment:
But `req.user` is the whole user object and `postBy` is expecting only the user id